### DISCUSSION TITLE
Can we run openlit ui on different port

### DISCUSSION THREAD

**Discussion by aravinth-numberz** [openlit/openlit#383] [General]
Is there a way to run openlit UI on different port instead of 3000 with docker compose ?

**Comment by amanagarwal042** [comment]
You can always change the port that the OpenLIT container runs on in your system.

**Reply by aravinth-numberz** [reply]
ok, thanks for the update @patcher9

**Reply by Valadaress** [reply]
Hello, how can i do it? I changed every reference inside the code of 3000 and set 3001 for example: `ENV OPENLIT_URL=http://127.0.0.1:3001` inside Dockerfile, I changed the `NEXTAUTH_URL=http://localhost:3001/` but next.js is still in the end serving on 3000

![image](https://github.com/user-attachments/assets/496e73d1-862b-4e3b-8f29-d6a0c18c91c0)
